frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/CreateProperty/CorrespondenceAddressSection.js

```
@@ -0,0 +1,96 @@
+        value={correspondenceAddress}
+        onChange={handleChange}
+        disabled={sameAsProperty}
+        placeholder={t("Enter")}
+      />
+      <div style={checkboxContainer}>
+        <input
+          type="checkbox"
+          checked={sameAsProperty}
+          onChange={handleCheckboxChange}
+        />
+        <label>{t("Same as property address")}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)